Dockerfile

```diff
@@ -1,30 +1,35 @@
-FROM alpine:latest
+FROM ubuntu:bionic
+
+ENV CANON_DRIVER_URL='http://gdlp01.c-wss.com/gds/8/0100007658/08/linux-UFRII-drv-v370-uken-05.tar.gz'
 ENV CUPS=1.0
 
+# Add repos
+RUN echo 'deb http://us.archive.ubuntu.com/ubuntu/ bionic multiverse' >> /etc/apt/sources.list.d/multiverse.list && \
+	echo 'deb-src http://us.archive.ubuntu.com/ubuntu/ bionic multiverse' >> /etc/apt/sources.list.d/multiverse.list && \
+	echo 'deb http://us.archive.ubuntu.com/ubuntu/ bionic-updates multiverse' >> /etc/apt/sources.list.d/multiverse.list && \
+	echo 'deb-src http://us.archive.ubuntu.com/ubuntu/ bionic-updates multiverse' >> /etc/apt/sources.list.d/multiverse.list && \
+	echo 'deb http://archive.ubuntu.com/ubuntu/ bionic-security multiverse' >> /etc/apt/sources.list.d/multiverse.list && \
+	echo 'deb-src http://archive.ubuntu.com/ubuntu/ bionic-security multiverse' >> /etc/apt/sources.list.d/multiverse.list
+
 # Install the packages we need. Avahi will be included
-RUN echo "http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories &&\
-	apk add --update cups \
-	cups-libs \
+RUN apt-get update && apt-get install -y \
+	brother-lpr-drivers-extra brother-cups-wrapper-extra \
+	cups \
 	cups-pdf \
-	cups-client \
-	cups-filters \
-	cups-dev \
-	gutenprint \
-	gutenprint-libs \
-	gutenprint-doc \
-	gutenprint-cups \
-	ghostscript \
-	avahi \
 	inotify-tools \
-	python3 \
-	python3-dev \
-	py3-pip \
-	build-base \
-	wget \
-	rsync \
-	&& pip3 --no-cache-dir install --upgrade pip \
-	&& pip3 install pycups \
-	&& rm -rf /var/cache/apk/*
+	libglade2-0 \
+  libpango1.0-0 \
+  libpng16-16 \
+	python-cups \
+	curl \
+&& rm -rf /var/lib/apt/lists/*
+
+# Install UFRII drivers
+RUN curl $CANON_DRIVER_URL | tar xz && \
+    dpkg -i *-UFRII-*/64-bit_Driver/Debian/*common*.deb && \
+    dpkg -i *-UFRII-*/64-bit_Driver/Debian/*ufr2*.deb && \
+    dpkg -i *-UFRII-*/64-bit_Driver/Debian/*utility*.deb && \
+    rm -rf *-UFRII-*
```

Comment:
Until this change, it was possible to build the image also for armv7 and arm64. The Raspberry Pis are a great platform for an airprint-server.

Are there no alternatives, which can be installed by the package manager?